### ISSUE TITLE
cannot use docker cli after docker-desktop removal on linux mint

### ISSUE THREAD

**Issue by ralusnom** [docker/desktop-linux#7]
TL;DR
After I removed Docker desktop, I can no longer use the usual Docker CLI.

  - [x] I have tried with the latest version of Docker Desktop
  - [x] I have tried disabling enabled experimental features
  - [ ] I have uploaded Diagnostics. _No. Where's that in docker desktop?_

### Expected behavior
Docker cli is usable

### Actual behavior
It aborts at the command line with the error message below. No matter what docker command I type in.
```
$ docker stats
Cannot connect to the Docker daemon at unix:///home/User/.docker/desktop/docker.sock. Is the docker daemon running?
```

I found that apt remove docker-desktop show the following message:
```Could not disable the Docker Desktop service. If it is still enabled, it will have to be disabled manually.
rm: cannot remove '/home/User/.local/share/systemd/user/docker-desktop.service': No such file or directory
```

### Information

  - Linux distro: Linux Mint 20.3 Cinnamon
  - Distro version: focal
  - Docker Desktop Version: https://desktop-stage.docker.com/linux/main/amd64/77103/docker-desktop.deb 

I've already tried a few things, but I can't get Docker to work anymore. Deleted everything, reinstalled. No matter what I do, I always get the same error.

I had Docker Engine on it first, then Docker Desktop. Actually I didn't use Docker desktop and wanted to uninstall it again. That worked, but now I can't use the cli anymore as noted above.

Apart from that all the actual Docker service is running as I can use it from my IDE (Intellij).

### Steps to reproduce the behavior

Open commendline and try to stat docker. 

  1. Install docker 
  2. Install docker-desktop
  3. Remove docker-desktop (via apt)
  4. Type `docker ps` or `docker stats` or ...

**Comment by p1-0tr** [comment]
Looks like your docker context was not restored to `default`. Desktop should normally do that, but if something went wrong when uninstalling, which the remove log suggests, that may not have been handled correctly.

You can change your context to `default` with - `docker context use default`, or run a single command with a specified context with `docker -c <context name> <command>` (https://docs.docker.com/engine/context/working-with-contexts/).
